flow-server/src/main/java/com/vaadin/flow/server/auth/DefaultNavigationCheckDecisionResolver.java

```diff
@@ -45,6 +45,21 @@
  * | ALL NEUTRAL     | DENY     |
  * | ALLOW + DENY    | REJECT   |
  * </pre>
+ *
+ * <p>
+ * </p>
+ * Almost the same rule applies also if the evaluation happens during error
+ * handling phase ({@link NavigationContext#isErrorHandling()} is
+ * {@literal true}), with a single exception: in this case, if all the results
+ * are {@literal NEUTRAL} the access is granted because the target of the
+ * navigation is supposed to be an error handler component and not a view with
+ * sensible information.
+ * <p>
+ * </p>
+ * To be noted that the above situation never happens if the
```

Comment:
Yours is also grammatically correct, but for me, this expression is a bit more readable:
```suggestion
 * It should be noted that the above situation never occurs if the
```

Reply:
It looks better. Thank you